README.md

```diff
@@ -33,7 +33,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
 
-    - uses: azure/login@v1.1
+    - uses: azure/login@v1.1.1
```

Comment:
@Fokko while using the action, use the major version so that you will get the latest updated version. Eg: azure/login@v1. Here in documentation, we will update it to point major version.